core/src/packages/unstable/blake2s.rs

```diff
@@ -42,7 +43,9 @@ use snarkos_models::{
     },
 };
 
-pub const CORE_UNSTABLE_BLAKE2S_NAME: &str = "Blake2s";
+// internal identifier
+pub const CORE_UNSTABLE_BLAKE2S_NAME: &str = "#blake2s";
+pub const CORE_UNSTABLE_BLAKE2S_PACKAGE_NAME: &str = "Blake2s";
```

Comment:
Just leaving a remark that we may want to have a standard library associating names to circuits and names to AST components, as matching on strings is a bit weak.

Reply:
I plan to move this into a later stage, this is just to hold Blake2s over without adding a custom AST node for it. I'll clean it up a bit.

The proposal I have in mind is a generic FFI binding scheme.